examples/first-class-constraints-and-fixpoints.flix

```diff
@@ -1,22 +1,22 @@
 /// Declare two predicate symbols.
-rel ParentOf(x: String, y: String)
-rel AncestorOf(x: String, y: String)
+//rel ParentOf(x: String, y: String)
+//rel AncestorOf(x: String, y: String)
 
 /// Returns a collection of facts.
-def getFacts(): #{ ParentOf, AncestorOf } = #{
-    ParentOf("Pompey", "Strabo").
-    ParentOf("Gnaeus", "Pompey").
-    ParentOf("Pompeia", "Pompey").
-    ParentOf("Sextus", "Pompey").
-}
+///def getFacts(): #{ ParentOf, AncestorOf } = #{
```

Comment:
For example here, you should change ParentOf to `ParentOf(String, String)` because you have deleted the ParentOf alias above.